formidling/src/main/resources/pdfgen/templates/ungdomsytelse/endring_programperiode.hbs

```diff
@@ -0,0 +1,34 @@
+{{#> felles/base}}
+
+    {{#> felles/seksjon/hoved }}
+        <h1>Vi har endret ungdomsprogramytelsen din</h1>
+
+        {{#with endretStartdato}}
+            <p>Fra {{nyDato}} får du penger gjennom ungdomsprogramytelse.</p>
+            <p>
+                Du fikk tidligere beskjed om at du skulle få ungdomsprogramytelse fra og med {{tidligereDato}},
+                men den datoen gjelder ikke lenger fordi den er endret av din veileder.
+                Derfor har du nå fått en ny dato for når ungdomsprogramytelse din starter.
+            </p>
+        {{/with}}
+        {{#with endretSluttdato}}
+            <p>Fra {{nyDato}} får du ikke lenger penger gjennom ungdomsprogramytelse.</p>
```

Comment:
Antar vi her at sluttdato alltid endres til å være tidligere? Eg synes ikkje dette klinger så bra om sluttdato er endret lenger fram i tid.

Reply:
Nei dette gjelder for begge, men godt poeng! Det er sånn de har skissert det, men jeg kan gjøre sånn at `lenger` kun kommer når sluttdato flyttes bakover. Det blir sikkert tilpasset etter test

Reply:
jeg tror jeg venter med å endre til de har testet. Det er mulig det beste er å fjerne lenger permanent. Spiller det tilbake